pkg/commands/import/import.go

```diff
@@ -78,57 +84,45 @@ cat dependencies.yaml | kp import -f -`,
 
 			configHelper := k8s.DefaultConfigHelper(cs)
 
-			descriptor, err := getDependencyDescriptor(cmd, filename)
+			kpConfig, err := configHelper.GetKpConfig(ctx)
 			if err != nil {
 				return err
 			}
 
-			repository, err := configHelper.GetCanonicalRepository(ctx)
+			imgFetcher := rup.Fetcher(tlsConfig)
+			imgRelocator := rup.Relocator(ch.Writer(), tlsConfig, ch.CanChangeState())
+
+			importer := importpkg.NewImporter(
```

Comment:
Do the tests for this invoke the real importer?

Reply:
ya we just pass in a fake util provider

Reply:
In that case does this even need an interface for the importer?

Reply:
i hadn't really thought about it but i guess it doesn't